exonum/src/runtime/types.rs

```diff
@@ -470,20 +473,23 @@ impl ProtobufConvert for ArtifactStatus {
     type ProtoStruct = schema::lifecycle::ArtifactState_Status;
 
     fn to_pb(&self) -> Self::ProtoStruct {
+        use self::schema::lifecycle::ArtifactState_Status::*;
+
         match self {
-            ArtifactStatus::Active => schema::lifecycle::ArtifactState_Status::ACTIVE,
-            ArtifactStatus::Pending => schema::lifecycle::ArtifactState_Status::PENDING,
+            ArtifactStatus::Active => ACTIVE,
+            ArtifactStatus::Pending => PENDING,
+            ArtifactStatus::Unloading => NONE,
             ArtifactStatus::__NonExhaustive => unreachable!("Never actually generated"),
         }
     }
 
     fn from_pb(pb: Self::ProtoStruct) -> Result<Self, failure::Error> {
+        use self::schema::lifecycle::ArtifactState_Status::*;
+
         Ok(match pb {
-            schema::lifecycle::ArtifactState_Status::ACTIVE => ArtifactStatus::Active,
-            schema::lifecycle::ArtifactState_Status::PENDING => ArtifactStatus::Pending,
-            schema::lifecycle::ArtifactState_Status::NONE => {
-                bail!("Status `NONE` is reserved for the further usage.")
-            }
+            ACTIVE => ArtifactStatus::Active,
+            PENDING => ArtifactStatus::Pending,
+            NONE => ArtifactStatus::Unloading,
```

Comment:
Is that correct? Either a new variant shall be added to the message, or NONE renamed to 'unloading'.

Reply:
I've renamed `NONE` -> `UNLOADING`.

Reply:
Looks good, thanks.